packages/components/src/styles/components/app-header.scss

```diff
@@ -106,16 +106,11 @@
   }
 
   // Dropdown & Button color theming overrides
+
   .hds-button,
-  .hds-dropdown-toggle-button,
-  .hds-dropdown-toggle-icon {
-    // Apply dark theme to child interactive components not within a nested dropdown
-    &:not(
-      .hds-dropdown * *,
-      .ember-basic-dropdown-trigger * *,
-      .ember-basic-dropdown-content * *,
-      .hds-dialog-primitive__wrapper *
-    ) {
+  button[class*="hds-dropdown-toggle"] {
+    // Apply dark theme to child interactive components not within a nested content container
+    &:not(.hds-dropdown__content *, .ember-basic-dropdown-content *, .hds-dialog-primitive__wrapper *) {
```

Comment:
I made the selectors more targeted to the content areas @didoo

Reply:
would `.hds-dropdown-toggle-button` work as well, instead of `button[class*="hds-dropdown-toggle"]`?